js/src/wp-seo-featured-image.js

```diff
@@ -160,6 +161,30 @@ import a11ySpeak from "a11y-speak";
 		if ( "undefined" !== typeof $featuredImageElement.prop( "src" ) ) {
 			featuredImagePlugin.setFeaturedImage( $( "#set-post-thumbnail " ).html() );
 		}
+
+		// Fallback for Gutenberg, as the featured image id does not exist there.
+		let imageData;
+		if ( isGutenbergDataAvailable() ) {
```

Comment:
I'd consider inverting this if and return early in case the Gutenberg data is not available.